gammapy/modeling/models/spatial.py

```diff
@@ -1282,3 +1286,129 @@ def plot_interactive(self, ax=None, geom=None, **kwargs):
         if geom is None:
             geom = self.map.geom
         super().plot_interactive(ax=ax, geom=geom, **kwargs)
+
+
+class PiecewiseNormSpatialModel(SpatialModel):
+    """Piecewise spatial correction
+       with a free normalization at each fixed nodes.
+
+       For more information see :ref:`piecewise-norm-spectral`.
```

Comment:
This reference does not exist yet? Add in follow up PR?

Reply:
added